card/mod/05_standard/set/type/search_type.rb

```diff
@@ -231,13 +250,6 @@ def default_search_params
     set_default_search_params :default_limit=>20
   end
   
-  
 end
 
 
```

Comment:
lots of these changes are from my moving stuff in from bootstrap mod.

In general, if it calls "super" I leave it, because it's actually separating things out.  But if it's a hard override, I usually just copy it back in; no need to keep two copies of the same code.